server/index.js

```diff
@@ -1,15 +1,19 @@
+require('newrelic');
 const express = require('express');
 
 const app = express();
 const port = 3100;
 const morgan = require('morgan');
 const bodyParser = require('body-parser');
 const cors = require('cors');
+const compression = require('compression');
 const dbpsql = require('./db/index-psql.js');
 // const dbcql = require('./db/index-cql.js');
 
+app.use(compression());
 app.use(cors());
-app.use('/listing/:id', express.static('public'));
+// app.use('/listing/:id', express.static('public')); //uncomment when only using this service and not the proxy
+app.use(express.static('public'));
```

Comment:
When using proxy, what's the URL you type into the browser? Doesn't have the /listing?

Reply:
Yup, correct. The proxy will using /listing/:listingID, which will then serve up the index.html, which will have the script tag containing the url source of my service, thus serving up my service's bundle.js file